test/integration/targets/includes/runme.sh

```diff
@@ -11,3 +11,6 @@ set +e
 result="$(ansible-playbook -i ../../inventory include_on_playbook_should_fail.yml -v "$@" 2>&1)"
 set -e
 grep -q "ERROR! 'include' is not a valid attribute for a Play" <<< "$result"
+
+ansible-playbook includes_loop_rescue.yml --extra-vars strategy=linear "$@"
+ansible-playbook includes_loop_rescue.yml --extra-vars strategy=free "$@"
```

Comment:
`StrategyBase._load_included_file()` updates `tqm._stats` but `IncludeRole.get_block_list()` does not so there's a difference between stat reporting for included files vs included roles. Maybe that should be moved outside of the loop too, where hosts are marked as failed for the PlayIterator?

These tests could be updated to check the final rescued count. On this branch the recap is currently:
```
PLAY RECAP *********************************************************************
localhost                  : ok=4    changed=0    unreachable=0    failed=2    skipped=0    rescued=0    ignored=0 
```

Looks good to me other than that.

Reply:
Oh, I didn't remember I mentioned it somewhere else - I had a pending review that I never published on this PR. My reasoning for fixing `get_block_list` specifically is since a failed include_role is fatal to the playbook the stats were fine on devel. But it didn't seem like an easy fix when I looked at it initially and since there's already the issue with failed included tasks, fixing separately works for me too.